src/pages/en.md

```diff
@@ -335,7 +335,7 @@ When should I bump a version?
 
 It's a good practice to publish RC/beta versions prior to releasing a major update.
 
-You can automate your release process by using, for example, [`gem-release`](https://github.com/svenfuchs/gem-release) or CI services (e.g. [Travis supports](https://docs.travis-ci.com/user/deployment/rubygems/) RubyGems deployments).
+You can automate your release process by using, for example, [`Gemsmith`](https://github.com/bkuhlmann/gemsmith) or CI services (e.g. [Travis supports](https://docs.travis-ci.com/user/deployment/rubygems/) RubyGems deployments).
```

Comment:
Let's not remove `gem-release` but mention both tools (i.e. "for example, Gemsmith or gem-release").